internal/testhelpers/selfservice_verification.go

```diff
@@ -43,9 +45,14 @@ func GetRecoveryFlow(t *testing.T, client *http.Client, ts *httptest.Server) *kr
 	require.NoError(t, err)
 	require.NoError(t, res.Body.Close())
 
-	rs, _, err := publicClient.V0alpha2Api.GetSelfServiceRecoveryFlow(context.Background()).Id(res.Request.URL.Query().Get("flow")).Execute()
-	require.NoError(t, err, "%s", res.Request.URL.String())
-	assert.Empty(t, rs.Active)
+	flowID := res.Request.URL.Query().Get("flow")
+	assert.NotEmpty(t, flowID, "expected to receive a flow id, got none")
+
+	rs, _, err := publicClient.V0alpha2Api.
+		GetSelfServiceRecoveryFlow(context.Background()).
+		Id(flowID).
+		Execute()
+	require.NoError(t, err, "expected no error when fetching recovery flow: %s", err)
```

Comment:
Please respect existing assertions:

```
	assert.Empty(t, rs.Active)
```

Reply:
We are now setting the active strategy when the flow is initialized. So the assertion needs to be inverted. 

Btw.: not sure if that's the best way, but we need a way to distinguish between link & code. Suggestions welcome.

Reply:
Done, by flipping the assertion, because now the field is always set.